packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.props.ts

```diff
@@ -8,12 +8,25 @@ import {
  * Scroll interface for consumers
  * 'start' is when the horizontal overflow scroll is all the way left in LTR (all the way right in RTL)
  * 'end' is when the horizontal overflow scroll is all the right in LTR (all the way left in RTL)
+ * TODO #1177: Change interface property names to be more meaningful
  */
-export interface ScrollChange {
```

Comment:
You need to keep the interface name the same, otherwise anyone import this will get a module resolution error.

Reply:
I mentioned this to @janechu but she said that interface name weren't contractual, but only the properties themselves...

Reply:
+1 on deprecate and export a new one.

Reply:
Added